### DISCUSSION TITLE
New Branding

### DISCUSSION THREAD

**Discussion by artifishvr** [spongedsc/SpongeChat#64] [Q&A] [answered]
SpongeChat is kinda generic, SpongeGPT implies it uses a model from the GPT line, which isn't true anymore, and SpongeLLM implies it's a large language model. 
The "Logo" of spongechat should also be changed, for hopefully obvious reasons.

**Comment by Butterroach** [comment]
name it about:blank

**Comment by hyd3rs** [comment]
doodleass

**Comment by hyd3rs** [comment]
coming back to this, i think this would actually be much more appropriate to put into a discussion

**Comment by artifishvr** [comment]
done c:

**Comment by artifishvr** [comment]
doodlepants

**Comment by artifishvr** [comment]
# uwupilot

## reasoning:

- plays off of microsoft's "copilot" (bing chat)
- uwu - represents how the "spongeass" default model speaks
- pilot - the bot "pilots" multiple models and technologies

**Reply by hyd3rs** [reply]
the copilot name is so overused

**Reply by lordsirggg** [reply]
Fart pilot

**Reply by artifishvr** [reply]
> the copilot name is so overused

ok but thats funny

**Comment by artifishvr** [comment]
> the copilot name is so overused

ok what similar length word should replace pilot for the like conductor of all the different models

**Reply by lordsirggg** [reply]
Fart

**Reply by hyd3rs** [reply, marked as answer]
pathways

**Reply by hyd3rs** [reply]
personally i like pathways

**Reply by artifishvr** [reply]
oooooooo sounds hot

**Comment by artifishvr** [comment]
![uwupilot](https://github.com/spongedsc/SpongeChat/assets/59352535/2ed2009a-084b-4dba-9cbf-143ed03f4648)

logo (really good)
meaning: uuuh i messed up my chin really badly and it made a mark like that (but red ofc) and i thought it looked cool

**Reply by lordsirggg** [reply]
Bad logo

**Reply by artifishvr** [reply]
![91ba1c357159dd52df6488ac856b6cbb](https://github.com/spongedsc/SpongeChat/assets/59352535/7579951a-dd85-45c5-b47a-7708b1d6309c)

**Comment by artifishvr** [comment]
UwUPathways

**Comment by artifishvr** [comment]
PathUwUays

**Comment by artifishvr** [comment]
pathways on its own is cool too

**Reply by hyd3rs** [reply]
real
